includes/Admin/Enhanced_Catalog_Attribute_Fields.php

```diff
@@ -47,7 +47,7 @@ class Enhanced_Catalog_Attribute_Fields {
 	 */
 	private $category_handler;
 
-	public function __construct( $page_type, \WP_Term $term = null, \WC_Product $product = null ) {
+	public function __construct( $page_type, ?\WP_Term $term = null, ?\WC_Product $product = null ) {
```

Comment:
Implicitly marking parameter $term as nullable is deprecated, the explicit nullable type must be used instead

This change ensures that the parameters $term and $product can explicitly accept null values.